sdk/storage/perf-tests/storage-blob/README.md

```diff
@@ -21,3 +21,5 @@
      - `npm run perf-test:node -- CoreHTTPDownloadWithSASTest --warmup 2 --duration 7 --iterations 2 --parallel 2`
    - download using sas with core-rest-pipeline
      - `npm run perf-test:node -- CoreHTTPSDownloadWithSASTest --warmup 2 --duration 7 --iterations 2 --parallel 2`
+   - download test with profiling
+     - `npm run perf-test:node -- StorageBlobDownloadTest --duration 10 --profile true --parallel 64` 
```

Comment:
Do you need to use `--profile true`, or can you just use `--profile` to enable profiling?  Boolean command-line options usually don't require a `true/false` value.  Setting the option is `true`, and omitting the option is `false`.

Reply:
@HarshaNalluru: Do you know if we use `--profile` instead of `--profile true`?

Reply:
Both work fine.
Updated `--profile true` to `--profile` however in the example commands.